src/Service/Proxy.php

```diff
@@ -281,7 +281,7 @@ private function decodeMetadata(string $package): array
     {
         /** @var Metadata $metadata */
         $metadata = $this->metadata($package)->getOrElse(Metadata::fromString('[]'));
-        $metadata = \json_decode((string) \stream_get_contents($metadata->stream()), true);
+        $metadata = \json_decode((string) \stream_get_contents($metadata->stream()), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
Change in behaviour. Either try-catch or skip the throw enforcing rule. There is an `is_array` check just below it.